src/rprblender/export/mesh.py

```diff
@@ -270,6 +270,10 @@ def sync_visibility(rpr_context, obj: bpy.types.Object, rpr_shape: pyrpr.Shape,
     obj.rpr.export_visibility(rpr_shape, indirect_only)
     obj.rpr.export_subdivision(rpr_shape)
 
+    if is_emissive_object(obj):
```

Comment:
As we've discussed - not needed, it was the effect of Ray Visibility settings difference between RPR and Cycles. Separate PR for that ray visibility changes would be better.